spec/support/test_top_container.rb

```diff
@@ -7,7 +7,8 @@ class TestTopContainer < ActiveRecord::Base
     {
       scope:          :address,
       case_sensitive: false,
-      message:        'is already present in top_container'
+      message:        'is already present in top_container',
```

Comment:
I think we should change the key to `error_message` and for parent association to `parent_error_message`. Mention this in CHANGELOG.